live-examples/html-examples/content-sectioning/css/h1-h6.css

```diff
@@ -0,0 +1,20 @@
+@font-face {
+    font-family: 'Fira Sans';
+    src: local('FiraSans-Regular'),
+    url('/media/fonts/FiraSans-Regular.woff2') format('woff2');
+}
+
+h1 {
+    margin-top: 2px;
+    margin-bottom: 3px;
```

Comment:
You don't need this, if you're also setting `margin-bottom` on the H1 at line 13.